### DISCUSSION TITLE
Using hidden inputs as pseudo-state storage?

### DISCUSSION THREAD

**Discussion by Aryaman73** [react-hook-form/react-hook-form#6101] [Q&A] [answered]
I was wondering if it was a smart idea to use hidden inputs and react-hook-form to pass information across components. 

 I'm in a situation where I need to add some logic at a child component (using `useFormContext()`) which needs to be accessed by the parent component which has the actual form (`useForm()`). One solution that came to mind was creating and registering a hidden input (`<input type="hidden" ... />`) then using `setValue` and `getValue/watch`.
 
  Is there a simpler alternative I'm missing?

**Comment by bluebill1049** [comment, marked as answer]
`register('test', { value: 'whatever' })`

**Reply by Aryaman73** [reply]
🤦 that makes sense, thanks!
